app/views/main.scala.html

```diff
@@ -25,11 +26,32 @@
         <link rel="stylesheet" media="screen" href="@routes.Assets.versioned("stylesheets/manageuser.css")">
         <link rel="stylesheet" media="screen" href="@routes.Assets.versioned("stylesheets/restrictfeedbackpopup.css")">
         <link rel="stylesheet" media="screen" href="@routes.Assets.versioned("stylesheets/report.css")">
+        <link rel="stylesheet" media="screen" href="@routes.Assets.versioned("stylesheets/sessioncontent.css")">
         <script src="@routes.Assets.versioned("plugins/jquery.js")" type="text/javascript"></script>
         <script src="@routes.Assets.versioned("plugins/bootstrap.min.js")" type="text/javascript"></script>
         <script src="@routes.Assets.versioned("javascripts/header.js")" type="text/javascript"></script>
         <script type="text/javascript" src="@routes.JavascriptRouter.jsRoutes"></script>
     </head>
+    <div id="fb-root"></div>
```

Comment:
Why is javascript added on template? Can we move it to a js file and include that js file on the template?